.gitlab/functional_test/regression_detector.yml

```diff
@@ -1,7 +1,7 @@
 single-machine-performance-regression_detector:
   stage: functional_test
   rules:
-    - !reference [.except_mergequeue]
+    - !reference [.except_main_or_release_branch]
```

Comment:
Could the previous behavior of not running on merge queues be preserved along with this change? Testing aggregate changes in merge queues is something we'd want to opt into more intentionally.

Reply:
Oh right, I should have made this more explicit. I haven't reproduced the `.except_mergequeue` rule block here because it is already part of the `.except_main_or_release_branch` rule block:

https://github.com/DataDog/datadog-agent/blob/213682e97e1b6a265c00b46e6f66874d939d434a/.gitlab-ci.yml#L612-L617